spec/factories/broadcasts.rb

```diff
@@ -55,5 +55,11 @@
       type_of        { "Welcome" }
       processed_html { "Sloan here! 👋 I noticed that you haven't <a href='https://dev.to/t/explainlikeimfive'>asked a question</a> or <a href='https://dev.to/t/discuss'>started a discussion</a> yet. It's easy to do both of these; just click on 'Write a Post' in the sidebar of the tag page to get started!" }
     end
+
+    factory :download_app_broadcast do
+      title          { "Welcome Notification: download_app" }
+      type_of        { "Welcome" }
+      processed_html { "Sloan here, with one last tip! 👋  Have you downloaded the DEV mobile app yet? Consider <a href=’https://dev.to/downloads>downloading</a> it so you can access all of your favorite DEV content on the go!" }
```

Comment:
Same points re: the extra space, closing the quote, and using the right quotation mark here.